tests/logictest/Dockerfile

```diff
@@ -1,6 +1,7 @@
 FROM python:3.10.4-alpine3.15
 
 RUN /usr/local/bin/python -m pip install --upgrade pip && pip install --user mysql-connector six PyHamcrest requests environs fire
+RUN pip install https://github.com/youngsofun/clickhouse-sqlalchemy/archive/refs/heads/databend.zip
```

Comment:
also add to requirements.txt?

Reply:
It's essential to make our build reproducible. Is it possible to pin this package with hash? Constantly downloading from `head` could break our tests by coincidences.

Reply:
We can use this branch youngsofun/clickhouse-sqlalchemy/tree/databend  the same as main. After this pr merged, new changes will develop in other branches. merged after the function is stable.

Reply:
Negative.

Please install via `https://github.com/youngsofun/clickhouse-sqlalchemy/archive/dfd9ba3540dcb3c88f0850ef30746738bc6915f7.zip`